### PR TITLE
Add json schemas for pop token and identity qr codes

### PR DESCRIPTION
**Pull request by Warhedgehog**
In order to be able to verify the content of a scanned qr code, we need to add json schemas to protocol. This will mitigate error coming from user showing the wrong qr code to scan

### CODE REVIEW

**Review by Tyratox — APPROVED** [review]

File: protocol/qrcode/identity.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "resource:/protocol/qrcode/identity.json",
+    "title": "QRCode data for the user's main public key",
+    "description": "Match a data contained in a qrcode encoding the main public key",
+    "type": "object",
+    "additionalProperties": false,
+    "properties": {
+        "identity": {
```
Comment (pierluca) [inline protocol/qrcode/identity.json:9]: I feel we might regret later on calling this merely `identity` 😅
do you have any idea for a more unique name ?

Comment (Warhedgehog) [reply protocol/qrcode/identity.json:9]: It's purpose is to display the main public key and I don't think it should change so we can call it `main_public_key` ?

Comment (jbsv) [reply protocol/qrcode/identity.json]: or if may play the 🐫 , I would name it: `mainPublicKey`

Comment (Warhedgehog) [reply protocol/qrcode/identity.json]: I think we have more 🐍 than 🐫 in schemas especially in messageData ones. But I don't know what's preferred for the pop project 🙂

Comment (jbsv) [reply protocol/qrcode/identity.json]: Well, the standard http://json-schema.org/draft-07/schema# tells me that it should be 🐫. Mixing up the 2 is of course a bad idea. Maybe @pierluca has some historical stake on this ?

Comment (jbsv) [reply protocol/qrcode/identity.json]: OK, talked to @pierluca and it looks like the pop project has taken the 🐍 path tic this regard. So we should continue with it as a pop "standard".

**Review by jbsv — APPROVED** [review]

File: protocol/qrcode/identity.json (hunk 1 above)
Comment (Warhedgehog) [reply protocol/qrcode/identity.json]: Alright, thanks for the clarification !

**Pull request merged**